6-appsource/hello-world/go.mod

```
@@ -0,0 +1,3 @@
+module github.com/GoogleContainerTools/skaffold/examples/getting-started
+
+go 1.18
```

Let's use 1.22 if possible

```suggestion
go 1.22
```